src/ServiceControl.Audit/Program.cs

```diff
@@ -73,6 +77,33 @@ static Assembly TryLoadAssembly(AssemblyLoadContext loadContext, string folderPa
             return null;
         }
 
+        static void ReadExeConfiguration()
+        {
+            // ConfigurationManager on .NET is looking for {assembly}.dll.config files, but all previous versions of ServiceControl will have {assembly}.exe.config instead.
+            // This code reads in the exe.config files and adds all the values into the ConfigurationManager's collections.
+
+            var assembly = Assembly.GetExecutingAssembly();
+            var location = Path.GetDirectoryName(assembly.Location);
+            var assemblyName = assembly.GetName();
+            var exePath = Path.Combine(location, $"{assemblyName.Name}.exe");
```

Comment:
When executing locally with F5 the only thing I have is the DLL around and there is no exe. This change seems to break the F5 experience. Should we have some conditional logic that falls back to `dll.config`?

Reply:
On macOS, I have the following:

![image](https://github.com/Particular/ServiceControl/assets/1325611/302f391e-b9cf-4b8e-a2b2-a9dca2820aca)

The `ServiceControl` Unix executable, the `ServiceControl.dll`, and the corresponding `.dll.config` file.

Reply:
Shouldn't this code be guarded by an if checking for the `.exe.config` file existence?

Reply:
The issue is here is related to `{assemblyName.Name}.exe` being passed into `OpenExeConfiguration` not existing on non-Windows machines. I've changed the code to use `OpenMappedExeConfiguration`, which lets me pass in the actual config file name, and now this should work correctly on all OSes.